spark/src/main/java/org/opensearch/sql/spark/dispatcher/SparkQueryDispatcher.java

```diff
@@ -133,27 +136,17 @@ private String constructSparkParameters(String datasourceName) {
   private StartJobRequest getStartJobRequestForNonIndexQueries(
       DispatchQueryRequest dispatchQueryRequest) {
     StartJobRequest startJobRequest;
-    FullyQualifiedTableName fullyQualifiedTableName =
-        SQLQueryUtils.extractFullyQualifiedTableName(dispatchQueryRequest.getQuery());
```

Comment:
remove unused code in SQLQueryUtils.

Reply:
This part is removed. 
Do you mean `SQLQueryUtils.extractFullyQualifiedTableName` ? This is still used in IndexQueries for jobName creation and tags.